examples/ui-demo/src/utils/radius.ts

```diff
@@ -0,0 +1,15 @@
+// copied from aa-sdk values:
+// https://github.com/alchemyplatform/aa-sdk/blob/v4.x.x/account-kit/react/src/tailwind/components/border-vars.ts#L4
+export function getBorderRadiusValue(borderRadius: 'none' | 'sm' | 'md' | 'lg') {
```

Comment:
oh we should just export this as a utility from the `tailwind` package rather than copy pasting it